browser/brave_wallet/android/brave_wallet_native_worker.h

```diff
@@ -1,8 +1,8 @@
 /* Copyright (c) 2021 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
+ * This Source Code Form is subject to the terms of the Mozilla
+ * Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at
+ * http://mozilla.org/MPL/2.0/. */
```

Comment:
the license was right before the change

Reply:
that was accidental, fixing

Reply:
actually cl format does that to it and not sure why